### ISSUE TITLE
Replace property isExtinct by ConservationStatus

### ISSUE THREAD

**Issue by edgardmarx** [AKSW/life-knowledge-base#3]
Replace the property is extinct by a proper one ConservationStatus (https://en.wikipedia.org/wiki/Conservation_status#:~:text=The%20conservation%20status%20of%20a,extinct%20in%20the%20near%20future.), creating all types of conservation status i.e Data Deficitent (DD), NotEvaluated (NE) ... Extinct (EX)

**Comment by JLXE5** [comment]
Do you have a way in mind how to get a hold of this information? 
It does not seem to be contained in the original COL data (checked ColDP Archive and Darwin Core Archive).

**Comment by edgardmarx** [comment]
`select distinct ?s ?o  where {?s <http://rs.gbif.org/terms/1.0/isExtinct> ?o } LIMIT 100`

**Comment by JLXE5** [comment]
> `select distinct ?s ?o where {?s <http://rs.gbif.org/terms/1.0/isExtinct> ?o } LIMIT 100`

This gives me 
```
http://col.htwk-leipzig.de/325YB,False
http://col.htwk-leipzig.de/328MG,False
http://col.htwk-leipzig.de/328MS,False
http://col.htwk-leipzig.de/328MT,False
http://col.htwk-leipzig.de/328MX,False
http://col.htwk-leipzig.de/328MY,False
http://col.htwk-leipzig.de/32YBQ,False
http://col.htwk-leipzig.de/32YBR,False
http://col.htwk-leipzig.de/32YKK,False
http://col.htwk-leipzig.de/3337W,False
http://col.htwk-leipzig.de/3337X,False
http://col.htwk-leipzig.de/3337Y,False
http://col.htwk-leipzig.de/33382,False
http://col.htwk-leipzig.de/33384,False
http://col.htwk-leipzig.de/33389,False
http://col.htwk-leipzig.de/3338B,False
http://col.htwk-leipzig.de/3338C,False
http://col.htwk-leipzig.de/3338D,False
http://col.htwk-leipzig.de/3338F,False
http://col.htwk-leipzig.de/3338G,False
http://col.htwk-leipzig.de/3338H,False
http://col.htwk-leipzig.de/3338J,False
http://col.htwk-leipzig.de/3338K,False
http://col.htwk-leipzig.de/3338L,False
http://col.htwk-leipzig.de/3338M,False
http://col.htwk-leipzig.de/3338P,False
http://col.htwk-leipzig.de/3338Q,False
http://col.htwk-leipzig.de/3338R,False
http://col.htwk-leipzig.de/3338S,False
http://col.htwk-leipzig.de/3338T,False
http://col.htwk-leipzig.de/3338V,False
http://col.htwk-leipzig.de/3338X,False
http://col.htwk-leipzig.de/33393,False
http://col.htwk-leipzig.de/33394,False
http://col.htwk-leipzig.de/33395,False
http://col.htwk-leipzig.de/33396,False
http://col.htwk-leipzig.de/33397,False
http://col.htwk-leipzig.de/3339B,False
http://col.htwk-leipzig.de/3339C,False
http://col.htwk-leipzig.de/3339D,False
http://col.htwk-leipzig.de/3339F,False
http://col.htwk-leipzig.de/3339G,False
http://col.htwk-leipzig.de/3339J,False
http://col.htwk-leipzig.de/3339K,False
http://col.htwk-leipzig.de/3339L,False
http://col.htwk-leipzig.de/3339N,False
http://col.htwk-leipzig.de/3339P,False
http://col.htwk-leipzig.de/3339Q,False
http://col.htwk-leipzig.de/3339S,False
http://col.htwk-leipzig.de/3339V,False
http://col.htwk-leipzig.de/3339W,False
http://col.htwk-leipzig.de/3339X,False
http://col.htwk-leipzig.de/3339Z,False
http://col.htwk-leipzig.de/333B3,False
http://col.htwk-leipzig.de/333B4,False
http://col.htwk-leipzig.de/333B5,False
http://col.htwk-leipzig.de/333B6,False
http://col.htwk-leipzig.de/333B9,False
http://col.htwk-leipzig.de/333BC,False
http://col.htwk-leipzig.de/333BD,False
http://col.htwk-leipzig.de/333BH,False
http://col.htwk-leipzig.de/333BJ,False
http://col.htwk-leipzig.de/333BL,False
http://col.htwk-leipzig.de/333BM,False
http://col.htwk-leipzig.de/333BN,False
http://col.htwk-leipzig.de/333BP,False
http://col.htwk-leipzig.de/333BS,False
http://col.htwk-leipzig.de/333BT,False
http://col.htwk-leipzig.de/333BV,False
http://col.htwk-leipzig.de/333BW,False
http://col.htwk-leipzig.de/333BX,False
http://col.htwk-leipzig.de/333BZ,False
http://col.htwk-leipzig.de/333C2,False
http://col.htwk-leipzig.de/333C5,False
http://col.htwk-leipzig.de/333C7,False
http://col.htwk-leipzig.de/333C8,False
http://col.htwk-leipzig.de/333C9,False
http://col.htwk-leipzig.de/333CB,False
http://col.htwk-leipzig.de/333CC,False
http://col.htwk-leipzig.de/333CD,False
http://col.htwk-leipzig.de/333CF,False
http://col.htwk-leipzig.de/333CG,False
http://col.htwk-leipzig.de/333CH,False
http://col.htwk-leipzig.de/333CL,False
http://col.htwk-leipzig.de/333CM,False
http://col.htwk-leipzig.de/333CN,False
http://col.htwk-leipzig.de/333CP,False
http://col.htwk-leipzig.de/333CQ,False
http://col.htwk-leipzig.de/333CR,False
http://col.htwk-leipzig.de/333CS,False
http://col.htwk-leipzig.de/333CT,False
http://col.htwk-leipzig.de/333CX,False
http://col.htwk-leipzig.de/333CY,False
http://col.htwk-leipzig.de/333CZ,False
http://col.htwk-leipzig.de/333D2,False
http://col.htwk-leipzig.de/333D4,False
http://col.htwk-leipzig.de/333D5,False
http://col.htwk-leipzig.de/333D8,False
http://col.htwk-leipzig.de/333D9,False
http://col.htwk-leipzig.de/333DB,False
```
Please share your thoughts on how to extract more useful information out of this with me.

**Comment by edgardmarx** [comment]
Sure,

The provided query returns the entities (limited to 100) that contain the property `isExtinct` while the link provides all types of conservation status from species including extinct.
Your task consists of the following:

- Create a class called `ConservationStatus`
- Instantiate all the existing conservation status types here: https://en.wikipedia.org/wiki/Conservation_status
- Replace the property isExtinct by the property conservationStatus adding its respective value which in the case of the entities above is `Extinct`.

**Comment by JLXE5** [comment]
fixed with 1eb61611cdc8d49e23f0fa7637cdb40e1a9d63db.
